query/src/main/scala/filodb/query/PromCirceSupport.scala

```diff
@@ -116,8 +116,9 @@ object PromCirceSupport {
         timeSeriesScanned <- c.downField("timeSeriesScanned").as[Long]
         dataBytesScanned     <- c.downField("dataBytesScanned").as[Long]
         resultBytes     <- c.downField("resultBytes").as[Long]
+        partkeysVisited <- c.downField("partkeysVisited").as[Long]
       } yield {
-        QueryStatistics(group, timeSeriesScanned, dataBytesScanned, resultBytes)
+        QueryStatistics(group, timeSeriesScanned, dataBytesScanned, resultBytes, partkeysVisited)
```

Comment:
We already have `timeSeriesScanned` which can be used for this. We don't need a new stat

Reply:
ack, reverted this change